eox_core/edxapp_wrapper/backends/enrollment_h_v1.py

```diff
@@ -17,11 +17,13 @@
 from openedx.core.lib.exceptions import CourseNotFoundError
 from course_modes.models import CourseMode
 from rest_framework.exceptions import APIException
+from django.contrib.auth import get_user_model
 
 
 def create_enrollment(*args, **kwargs):
 
     errors = []
+    email = kwargs.get("email")
     user_id = kwargs.get('username')
```

Comment:
final nit: you assign username to the user_id. That is a bit confusing. Why not user `username = kwargs.get('username')`

Reply:
Changed. But the logic was because the functions we are calling (create_course_enrollment) always call it "user_id" in their definitions